tests/models/speecht5/test_modeling_speecht5.py

```diff
@@ -1021,15 +1028,65 @@ def test_generation(self):
         input_ids = processor(text=input_text, return_tensors="pt").input_ids.to(torch_device)
 
         generated_speech = model.generate_speech(input_ids, speaker_embeddings=speaker_embeddings)
-        self.assertEqual(generated_speech.shape, (228, model.config.num_mel_bins))
```

Comment:
Why did it change to `230` here ?

Reply:
On my branch it's always 230. Will check on that.

Reply:
Hi @ylacombe, are you sure the first dimension of `generated_speech.shape` is 228 here? I re-test it on my branch and also on the main branch of huggingface transformers, and they all get following error!

```
FAILED tests/models/speecht5/test_modeling_speecht5.py::SpeechT5ForTextToSpeechIntegrationTests::test_generation - AssertionError: torch.Size([230, 80]) != (228, 80)
```

The command I use to test is

```
RUN_SLOW=1 pytest tests/models/speecht5/test_modeling_speecht5.py::SpeechT5ForTextToSpeechIntegrationTests::test_generation
```

Meanwhile, I notice that the test `SpeechT5ForTextToSpeechIntegrationTests` have a decorator `@slow`  on it so I think  when merging your previous PR, this test is skipped so it does not raise an error!

Could you please have a re-check on that? For me I think here should be `self.assertEqual(generated_speech.shape, (230, model.config.num_mel_bins))`.